apps/emqx_durable_storage/src/emqx_ds_msg_serializer.erl

```diff
@@ -47,20 +47,26 @@ check_schema(v1) ->
     ok;
 check_schema(asn1) ->
     ok;
+check_schema(verbatim) ->
```

Comment:
Investigate removing msg_serializer from the TTV layouts. In the ideal world payload (de)serialization should happen on the frontend, not in the storage layout.